contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,80 @@
+---
+date: 2023-03-02
+title: "Introducing HogQL for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+author:
+  - joe-martin
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+---
+
+Today, we're excited to announce a major new feature which we're making available to all users as a public beta on PostHog Cloud. It's called HogQL and it's a new query language which makes it easier to interrogate your data directly.
+
+Not only that, but the introduction of HogQL brings with it other updates, including a new event explorer (previously the Live Events view), and the ability to use HogQL expressions within insights. We think that PostHog's existing insights will remain valuable for the vast majority of queries, but HogQL is an alternative when they just don't fit the bill.
+
+Speaking of bills, HogQL is free for all users while it's in public beta - all we ask is that you share any feedback you have with it either via the PostHog Slack, or by [filing an issue on our repo](https://github.com/PostHog/).
+
+## What is HogQL?
+
+HogQL is basically our take on SQL. 
+
+Less basically, it's a transition layer over ClickHouse SQL that we've created to empower a wide range of users, from technical product managers who need multi-property breakdowns, to engineers who need to dissect data in more nuanced ways. It's intuitive for teams who are familiar with SQL, but offers a few extra advantages and features.
+
+Unique features to HogQL include simplified access to event and person properties, as well as the ability to run subqueries and database joins. You can also use aggregations, such as `count`, `min`, `minIf`, `max`, `maxIf`, `sum`, `sumIf`, `avg`, `any`, and `anyIf`.
+
+We recommend checking the docs for a more detailed [explanation of ClickHouse SQL functions which are supported in HogQL](/manual/hogql).
+
+## HogQL expressions
+![HogQL trends breakdown filter](../images/features/hogql/trends-breakdown.png)
+
+A frequent request from teams using PostHog has been the ability to breakdown insight results across multiple properties. To accommodate this, we've added the ability to use HogQL expressions within an insight breakdown - so you can breakdown results however you like. 
```

Comment:
I don't think we have multiple breakdowns yet, just the ability to do more advanced breakdowns with HogQL

Reply:
@mariusandra Can you confirm? Disagrees with my notes from our chat.

Reply:
It's kind of the same thing if you know what you're doing. HogQL lets you create a combined "super-propety", even if just an array/tuple (`tuple(distinct_id, properties.bla)`) or a combined string of normal properties (`concat(properties.a, properties.b)`), and break down by that. This effectively serves the same purpose as the "multi breakdown interface" we could build.

We can now build a nicer interface around it that actually lets the user enter multiple values, and put them into a `tuple()` ourselves behind the hood.

There are other problems with HogQL breakdown filters. The biggest one is the UX bug that you can just add them, but you can't edit them. You have to delete the filter you added and make a new one. There's also some weird capitalisation that sometimes happens, etc.

It all needs some love, but technically, if you know what you're doing, you can break down by multiple properties :).